cdf-core/cdf/js-modules/addIns/colTypes/template/template.js

```diff
@@ -125,10 +125,10 @@ define([
         attachEvents: function($placeholder, events, info) {
           var myself = this;
           _.each(events, function(elem) {
-            var separator = ' ',
```

Comment:
@mfgaspar  @pamval Warning: the list of events to be attached is now comma-separated instead of being space-separated.
Does this have an impact on previous dashboards?